backend-forms.md

```diff
@@ -216,7 +216,7 @@ Option | Description
 ------------- | -------------
 **label** | a name when displaying the form field to the user.
 **type** | defines how this field should be rendered (see [Available fields types](#field-types) below). Default: text.
-**span** | aligns the form field to one side. Options: auto, left, right, full. Default: full.
+**span** | aligns the form field to one side. Options: auto, left, right, storm, full. Default: full. The parameter `storm` allows to form the form in the form of a Bootstrap grid, using the `cssClass` property, for example, `cssClass: col-xs-4`.
```

Comment:
`to form the form in the form of`: Mind simplifying that little ditty right there?

Reply:
@LukeTowers , yes, I fixed it